blocks/article-body-block/chains/article-body/_children/oembed.jsx

```diff
@@ -1,129 +1,25 @@
-
 import React from 'react';
-import PropTypes from 'prop-types';
-import EmbedContainer from 'react-oembed-container';
+import PropTypes from '@arc-fusion/prop-types';
 
 const Oembed = ({
   element,
-  // eslint-disable-next-line no-unused-vars
-  content,
-  isAmp,
-  isLeadArt,
-  // eslint-disable-next-line no-unused-vars
-  metaValue,
 }) => {
-  const responsiveClass = (['youtube', 'vimeo'].indexOf(element.subtype) !== -1 && !isAmp) ? 'embed-responsive embed-responsive-16by9' : '';
-
-  // Need to put this in a function outside return so we can do switching logic
-  const output = () => {
-    /* istanbul ignore if  */
-    if (isAmp) {
-      switch (element.subtype) {
-        case 'tweet':
-        case 'twitter':
-          const searchTerm = 'status/';
-          const tweetid = element.raw_oembed._id.substring(
-            element.raw_oembed._id.indexOf(searchTerm) + searchTerm.length,
-          );
-          return (
-            // eslint-disable-next-line no-useless-escape
-            <div className={`oembed | margin_centered oembed-${element.subtype} ${responsiveClass} flex-container-row\
-justify_center`}
-            >
-              <amp-twitter
-                width="300"
-                height="300"
-                layout="responsive"
-                data-tweetid={tweetid}
-              />
-            </div>
-          );
-        case 'instagram':
-          const instaRegex = /(https?:\/\/(www\.)?)?instagram\.com\/p\/(\w*([-'])?(\w*)?)/;
-          const instaId = element.raw_oembed
-            && element.raw_oembed.html
-            && element.raw_oembed.html.match(instaRegex);
-          if (instaId && instaId[3]) {
-            return (
-              <amp-instagram
-                data-shortcode={instaId[3]}
-                data-captioned
-                width="1"
-                height="1"
-                layout="responsive"
-              />
-            );
-          }
-          return null;
-
-        case 'youtube':
-          const decodedURL = element.raw_oembed._id.replace('%3D', '=');
-          const videoID = decodedURL.split('v=')[1];
-          return (
-            <amp-youtube
-              width="1.7"
-              height="1"
-              layout="responsive"
-              data-videoid={videoID}
-            />
-          );
-        case 'facebook':
-          return (
-            <amp-facebook
-              width={element.raw_oembed.width}
-              height="310"
-              layout="responsive"
-              data-href={element.raw_oembed.url}
-            />
-          );
-        case 'vimeo':
-          return (
-            <amp-vimeo
-              width="1.7"
-              height="1"
-              layout="responsive"
-              data-videoid={element.raw_oembed.video_id}
-            />
-          );
-        default:
-          return (
-            <EmbedContainer markup={element.raw_oembed.html}>
```

Comment:
remove embedcontainer as it's not being used for what it's intended for. see intended usecase: 

```jsx
render() {
  const { post } = this.props;
  return (
    <EmbedContainer markup={post.content.rendered}>

      {/* for example, */}
      <article id={`post-${post.id}`}>
        <h2>{ post.title.rendered }</h2>
        <div dangerouslySetInnerHTML={{ __html: post.content.rendered }} />
      </article>

    </EmbedContainer>
  );
}

```
https://github.com/humanmade/react-oembed-container

Reply:
not sure if removing this is part of the scope of this ticket, reverting